community/index.html

```diff
@@ -66,6 +66,15 @@
           icon="web_asset"
```

Comment:
Can you add a `continues=true` here so there is no separator within the Chat Room section?

Reply:
My bad. Fixed

![image](https://user-images.githubusercontent.com/41530422/99540919-5c1dd380-29da-11eb-8fcc-c84982165df8.png)